handlers/auth_code_request_create.go

```diff
@@ -63,6 +63,17 @@ func CreateAuthCodeRequest(authCodeReqSvc *service.AuthCodeRequestService) http.
 			request.OfficerUraID = officer.UsualResidentialAddress.ID
 			request.OfficerForename = officer.Forename
 			request.OfficerSurname = officer.Surname
+		} else {
+			// check if any eligible officers exist for specified company
+			companyIsEligible, err := service.CheckOfficers(request.CompanyNumber)
+			if err != nil {
+				utils.WriteErrorMessage(w, req, http.StatusInternalServerError, "there was a problem communicating with the Oracle API")
```

Comment:
Should probably log an error here as well

Reply:
`WriteErrorMessage` does logging (although perhaps it could be renamed, outside this PR)

Reply:
In that case we shouldn't use `WriteErrorMessage` on line 74 as returning no eligible officers isn't an error, just someone trying to use it who isn't eligible. And it might lead to red herrings when trying to debug through the logs. Should maybe use `log.Info` instead